.github/workflows/node-libs.yml

```
@@ -193,14 +199,14 @@ jobs:
       - name: Determine publish
         id: check-publish
         run: |
-          PUBLISHABLE=${{steps.check-branch.outputs.publishable == 'true' && steps.commit.outputs.publishable == 'true'}}
-          STORYBOOK_PUBLISHABLE=${{inputs.has-storybook == true && steps.commit.outputs.publishable == 'true'}}
+          PUBLISHABLE=${{ (inputs.skip-publish-branch-check || steps.check-branch.outputs.publishable == 'true') && steps.commit.outputs.publishable == 'true' }}
+          STORYBOOK_PUBLISHABLE=$([[ "${{ inputs.has-storybook }}" == "true" && "$PUBLISHABLE" == "true" ]] && echo "true" || echo "false")
```

```suggestion
          STORYBOOK_PUBLISHABLE=$([[ "${{ inputs.has-storybook }}" == "true" && "$PUBLISHABLE" == "true" ]] && (echo "true" || echo "false"))
```
What is used for the `echo "true" || echo "false"`

---

combining bools and ternery like to set `STORYBOOK_PUBLISHABLE`

---

got an idea, trying to simplify

---

updated